posthog/migrations/0210_drop_update_person_functions.py

```diff
@@ -0,0 +1,14 @@
+# Generated by Django 3.2.5 on 2022-02-11 15:43
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0209_plugin_logs_disabled"),
+    ]
+
+    operations = [
+        migrations.RunSQL("DROP FUNCTION IF EXISTS update_person_props, should_update_person_props"),
```

Comment:
Nit: I've just realized we made a typo here. The function is `should_update_person_prop`